tree/treeplayer/src/TTreeReaderValue.cxx

```diff
@@ -832,8 +832,8 @@ void ROOT::Internal::TTreeReaderValueBase::ErrorAboutMissingProxyIfNeeded()
 {
    // Print the error only if the branch name does not appear in the list of
    // missing proxies that the user explicitly requested not to error about
-   if (std::find(fTreeReader->fMissingProxies.cbegin(), fTreeReader->fMissingProxies.cend(), fBranchName.Data()) ==
-       fTreeReader->fMissingProxies.cend())
+   if (!fTreeReader || std::find(fTreeReader->fMissingProxies.cbegin(), fTreeReader->fMissingProxies.cend(),
+                                 fBranchName.Data()) == fTreeReader->fMissingProxies.cend())
       Error("TTreeReaderValue::Get()", "Value reader not properly initialized, did you call "
```

Comment:
We may wan to the branchname in the error message.

Reply:
Good point! I will add that in PR #18265 so the CI on this one doesn't need to rerun